src/main/java/org/isf/medicalstockward/gui/WardPharmacyNew.java

```diff
@@ -319,16 +321,16 @@ public boolean isCellEditable(int arg0, int arg1) {
 			return false;
 		}
 	}
-	
+
 	private boolean checkQuantityInLot(MedicalWard medWard, double qty) {
 		double wardQty = medWard.getQty();
 		if (qty > wardQty) {
-			MessageDialog.error(this, "angal.medicalstock.movementquantityisgreaterthanthequantityof.msg");
+			MessageDialog.error(this, "angal.medicalstock.movementquantityisgreaterthanthequantityof.fmt.msg");
```

Comment:
is this is a `fmt` type message, there should be some parameter/variable also to substitute into the message shouldn't there?

Reply:
good catch! yes, two parameteres... solved, thanks!